test/test_suite.sh

```diff
@@ -1,253 +1,276 @@
 #!./test/libs/bats/bin/bats
```

Comment:
The bat documentation recommends to use the `bats` extension:
https://github.com/bats-core/bats-core#usage